script/mailman_server

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+require "rubygems"
+require "bundler/setup"
+require "mailman"
+
+Mailman.config.logger = Logger.new("log/mailman.log")
+
+# Email Configration for mailman
+Mailman.config.pop3 = {
+  server: 'pop.example.com', 
```

Comment:
I guess setting the mail configuration is left.

Reply:
@Gauravano we can't explicitly write which service we are using....but we can use env variable instead of this.

Reply:
agreed!

Reply:
@icarito any recommendations on how to leave this configuration in this PR?